KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -161,15 +163,120 @@ end
 		Dλ1=obj.Dλ1,
 		Dλ2=obj.Dλ2,
 		Dθ=obj.Dθ,
+		motion=obj.motion
 		#Χ=obj.Χ[p], #TODO!
-		ux=obj.ux,
-		uy=obj.uy,
-		uz=obj.uz
 	)
 end
 
 """
-    obj = brain_phantom2D(; axis="axial", ss=4)
+dims = get_dims(obj)
+"""
+function get_dims(obj::Phantom)
+	dims = [0,0,0]
+	if obj.x != zeros(length(obj.x)) dims[1] = 1 end
+	if obj.y != zeros(length(obj.x)) dims[2] = 1 end
+	if obj.z != zeros(length(obj.x)) dims[3] = 1 end
+	dims
+end
```

Comment:
Is there a way to verify this without calculating `zeros(length(obj.x))`? Use explicit `return`s.